### PR TITLE
Add meeting minutes for 2017-07-06

### PR DESCRIPTION
**Pull request by bnb**
Adds minutes for the meeting held on July 6, 2017 (2017-07-06).

### CODE REVIEW

File: meetings/2017-07-06.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+## Links
+
+* **GitHub Issue**: [community-committee#87](https://github.com/nodejs/community-committee/issues/87)
+* **Minutes Google Doc**: https://docs.google.com/document/d/1utEp06Pyk0uwJn77TWdxIpwuoiFRjHvKinSdbY1BwNk/
+* _Previous Minutes Google Doc_:
+https://docs.google.com/document/d/1GXbfPhfbLTr7g9Hp7YP-M60ux_Ua2z8XRXg__me4u4k/
+* _Video Recording_:
+https://www.youtube.com/watch?v=04JNTd_UXZw
+## Present
```
Comment (nebrius) [inline meetings/2017-07-06.md]: Michael Dawson was also present, can you add him?

File: meetings/2017-07-06.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+**In-meeting votes:**
+* Tracy: +1
+* William: +1
+* Bryan: +1
+* Gregor: +1
+
+Voting will have to continue in the repo, in GitHub. Tracy [requested](https://github.com/nodejs/community-committee/pull/77#issuecomment-313516728) that members not present in the meeting vote.
+ 
+Discussion around 3 month limit on member inactivity, William clarified that this landed recently and is not applied retroactively.
```
Comment (nebrius) [inline meetings/2017-07-06.md]: Let's clarify this a bit. We don't have a limit on activity, but rather a requirement that members need to participate. If a member does not participate within a three month time period, then they are converted from a member to an observer.

File: meetings/2017-07-06.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+**Action Items:**
```
Comment (nebrius) [inline meetings/2017-07-06.md]: Let's add who is responsible for these actions items. I'm responsible for these.

File: meetings/2017-07-06.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+Gregor requested further context on travel funds, and Tracy shared the context of what the travel fund is and the need for collaboration across the TSC/CommComm
+
+The group discussed different possibilities for enabling cross-Committee collaboration. Some possibilities are nodejs/collaborators and an admin team or repo.
+
+Further discussion around this issue came up again later in the meeting. The group talked about how to contribute and add to the list, in addition to adding a bit more context to this issue.
+
+One topic the group talked about as a part of this was the new iteration of the Node.js website and the inactivity in the project. The group discussed some of the history surrounding this new iteration, and what possible next steps may be including opening an issue in the Website WG repo to discuss how to proceed and the possibility of getting funding from the foundation to help with associated costs.
+
+**Action Items:**
+  * Create an issue around enabling cross-Committee discussion and collaboration
```
Comment (nebrius) [inline meetings/2017-07-06.md]: IIRC, @hackygolucky is taking this action item on, correct? Can we add that info here?

File: meetings/2017-07-06.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+  * Create an issue in the CommComm or TSC main repo.
+  * Travel Fund needs to be pushed as a priority, as it’s important and needs to be resolved
+
+### events: Node.js Collaborator’s Summit
+
+Discussion around who is attending the collaborator’s summit, when the Collaborator's Summit will be held in relation to NINA 2017, and what the CommComm can be doing at the Collaborator’s summit.
+
+This link about the Summit was shared by Brian: https://github.com/nodejs/summit/issues/44
```
Comment (nebrius) [inline meetings/2017-07-06.md]: Nit: S/Brian/Bryan

File: meetings/2017-07-06.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+One topic the group talked about as a part of this was the new iteration of the Node.js website and the inactivity in the project. The group discussed some of the history surrounding this new iteration, and what possible next steps may be including opening an issue in the Website WG repo to discuss how to proceed and the possibility of getting funding from the foundation to help with associated costs.
+
+**Action Items:**
+  * Create an issue around enabling cross-Committee discussion and collaboration
+  * Create an issue in the CommComm or TSC main repo.
+  * Travel Fund needs to be pushed as a priority, as it’s important and needs to be resolved
+
+### events: Node.js Collaborator’s Summit
+
+Discussion around who is attending the collaborator’s summit, when the Collaborator's Summit will be held in relation to NINA 2017, and what the CommComm can be doing at the Collaborator’s summit.
```
Comment (pup) [inline meetings/2017-07-06.md]: maybe unravel NINA, it took me a second :P

**Comment by bnb** [comment]
Addressed comments/nits - please review if you've got some time 😄

**Pull request merged**